lib/src/bindings/flatbuffers.dart

```diff
@@ -80,8 +81,17 @@ class _Allocator extends fb.Allocator {
 
   @override
   void clear(ByteData data, bool _) {
-    _memset ??= DynamicLibrary.process().lookupFunction<
-        Void Function(Pointer<Void>, Int32, IntPtr), _dart_memset>('memset');
+    if (_memset == null) {
+      DynamicLibrary lib;
+      if (Platform.isWindows) {
+        // DynamicLibrary.process() is not available on Windows
+        lib = DynamicLibrary.open('msvcr100.dll');
```

Comment:
Where is this workaround coming from? Appears to be a C library. Does this require installation of Microsoft Visual C++ 2015 Redistributable (x64) like mentioned at https://docs.objectbox.io/java-desktop-apps#native-libraries? Should this be noted in a README?

The GitHub Windows instance has a bunch of versions installed, so I guess most are fine. https://github.com/actions/virtual-environments/blob/win19/20201210.0/images/win/Windows2019-Readme.md#microsoft-visual-c

Reply:
> Where is this workaround coming from?

Nowhere in particular, I've just searched what libary defines `memset` on windows. Turns out it's `mscvrt.dll` or `mscvr100.dll`. Yes it may be fragile if users don't have the lib... I'll add another fallback layer

Reply:
Maybe add this in a comment then?

Reply:
OK, also added a comment